tools/testing/gatekeeper/required-tests.yaml

```diff
@@ -6,7 +6,7 @@ required_tests:
   - Shellcheck required / shellcheck-required
   # TODO: cargo-deny-runner.yaml not yet treated as conditional
   - Cargo Crates Check Runner / cargo-deny-runner
-
+  - GHA security analysis / zizmor
```

Comment:
If the PR isn't a workflow change and therefore the job isn't triggered, how will this interact with the required test?

Reply:
I think the path restriction makes sense, so I wonder if we need to have a category of workflow checks (e.g. zizmor & action lint) that only are required when there is a workflow change?

Reply:
My issue is that this would only be supported by gatekeeper required tests, not GH UI required tests. GH would still expect (1) zizmor to run for each PR and (2) (I think) zizmor results to be collected for each PR.